lua/iswap/ui.lua

```diff
@@ -42,7 +42,7 @@ function M.prompt(bufnr, config, nodes, active_range, times)
   for i, node in ipairs(nodes) do
     local key = keys:sub(i, i)
     map[key] = node
-    ts_utils.highlight_node(node, bufnr, M.argts_ns, config.hl_selection)
+    -- ts_utils.highlight_node(node, bufnr, M.argts_ns, config.hl_selection) -- cant see anything with it
```

Comment:
Is this for when you try to pick an ancestor? Because all the ancestors will encapsulate each other like Matryoshka dolls? If so, add another argument to function and make this boolean toggled.

Reply:
Ok, I just bough it back (I noticed I can disable node highlight  from config). I disabled it just, because I did not like it :/